src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/AnonymizerEngine.cs

```diff
@@ -65,17 +68,38 @@ public static AnonymizerEngine CreateWithFileContext(string configFilePath, stri
         public ITypedElement AnonymizeElement(ITypedElement element, AnonymizerSettings settings = null)
         {
             EnsureArg.IsNotNull(element, nameof(element));
+            try
+            {
+                ElementNode resourceNode = ElementNode.FromElement(element);
+                return resourceNode.Anonymize(_rules, _processors);
+            }
+            catch (AnonymizerConfigurationErrorsException)
+            {
+                throw;
+            }
+            catch
+            {
+                if(_configurationManager.Configuration.processingErrors == ProcessingErrorsOption.Skip)
+                {
+                    return null;
```

Comment:
is there any way for customer to get the detail error message?

Reply:
They can  not get details if they skip the exception.